src/renderer/entities/proxy/lib/proxy-utils.ts

```diff
@@ -49,8 +49,18 @@ function isSameProxyGroup(oldGroup: NoID<ProxyGroup>, newGroup: NoID<ProxyGroup>
 }
 
 // TODO: Add i18n for wallet name
-function getProxiedName(accountId: AccountId, proxyType: ProxyType, addressPrefix?: number): string {
-  return `${proxyType} for ${toAddress(accountId, { prefix: addressPrefix, chunk: 6 })}`;
+function getProxiedName(
+  accountId: AccountId,
+  proxyType: ProxyType,
+  proxyVariant: ProxyVariant,
+  addressPrefix?: number,
+): string {
+  const address = toAddress(accountId, {
+    prefix: addressPrefix,
+    chunk: 6,
+  });
+
+  return `${proxyType} ${proxyVariant === ProxyVariant.PURE ? 'for pure' : 'for'} ${address}`;
```

Comment:
I guess having a variable for `proxyVarian` is a good idea, like you did for `address`